spec/git_reflow/all-approval/git_reflow_spec.rb

```diff
@@ -18,6 +18,11 @@
   let(:existing_pull_request)  { GitReflow::GitServer::GitHub::PullRequest.new existing_pull_requests.first }
 
   before do
+
```

Comment:
👍  for retaining the original file path for the original spec.  There is not much benefit to nesting it and it destroys the git history of the entire file from an `annotate` perspective.